compiler/src/codegen/transl_anf.re

```diff
@@ -112,7 +112,16 @@ let global_name = slot => Printf.sprintf("global_%d", slot);
 let find_id = (id, env) =>
   try(Ident.find_same(id, env.ce_binds)) {
   | Not_found =>
-    let (_, slot, alloc) = Ident.find_same(id, global_table^);
+    let (_, slot, alloc) =
+      try(Ident.find_same(id, global_table^)) {
+      | Not_found =>
+        Printf.eprintf("Not_found: %s\nKnown binds:\n", Ident.name(id));
```

Comment:
I added this for debugging. I was about to delete it just now, but maybe we want to leave it?

Reply:
It's not in a standalone commit. I'll just back it out.